### ISSUE TITLE
[strategic] Add power grid simulation

### ISSUE THREAD

**Issue by tupolov** [ALiVEOS/ALiVE.OS#227]
Identify power stations and pylons.

Link power stations and pylons to locations

Cut lighting in areas where power grid is disrupted.

**Comment by coachwn** [comment]
I have an sqf that I got which allows you to set markers for lighting and power stations. Will post it up when I get home.

**Comment by coachwn** [comment]
https://drive.google.com/file/d/0B6RjvmUcuDjgTDFZX1BBWHZkOGM/view?usp=drivesdk

**Comment by coachwn** [comment]
I can not remember where I got the file from but the guys were using it in a mission series on Altis.

**Comment by nateberkopec** [comment]
So hot on this. http://www.armaholic.com/page.php?id=15178 AEG was a wonderful mod.

**Comment by coachwn** [comment]
Yeah I loved AEG but it was dependent on map support first and foremost. Still was a great thing though.

**Comment by curiousfREAk** [comment]
I did one recently for Chernarus. I'll post it here, I'ts an working pre-Alpha, ... uncommented code, ... gimme two days to sort it out.

**Comment by coachwn** [comment]
Can't wait to see how that looks and works.

**Comment by curiousfREAk** [comment]
Update: Found it, .... don't understand my own work ^^. Rewriting it currently.

**Comment by curiousfREAk** [comment]
... found time Sunday afternoon. I try to get this "pack of Scripts" in something
more versatile then just a Chernarus Grid. So the good news: I'm on it. ;)

**Comment by curiousfREAk** [comment]
Update: The 3DEN Modules (where we can create Powergrids) is undergoing Alpha its first tests. Still there are some Screws to tighten but at least with the 3Ditor Modules, the biggest Part is coming to an end ...
.. even if my Head still is full of "wanahave-wannahave-wannahave-features" .... 
My next steps: 
- Server-Client implementation,
- Github-Project,
- Closed Beta Test in our Community.
- Public Beta
ETA .... hard to tell, ... giving my best.

cheers, fREAk
